src/current/v24.3/changefeed-best-practices.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Changefeed Best Practices
+summary: A selection of best practices for starting and running changefeeds.
+toc: true
+---
+
+This page describes best practices to consider when starting changefeeds on a CockroachDB cluster. We recommend referring to this information while planning your cluster's changefeeds and following the links in each of the sections for more details on a topic.
+
+## Plan the number of watched tables for a single changefeed
+
+When creating a changefeed, it's important to consider the number of changefeeds versus the number of tables to include in a single changefeed:
+
+- Changefeeds each have their own memory overhead, so every running changefeed will increase total memory usage.
+- Creating a single changefeed that will watch hundreds of tables can affect the performance of a changefeed by introducing coupling, where the performance of a target table affects the performance of the changefeed watching it. For example, any [schema change]({% link {{ page.version.version }}/changefeed-messages.md %}#schema-changes) on any of the tables will affect the entire changefeed's performance.
+
+To watch multiple tables, we recommend creating a changefeed with a comma-separated list of tables. However, we do **not** recommend creating a single changefeed for watching hundreds of tables.
+
+{% include {{ page.version.version }}/cdc/recommendation-monitoring-pts.md %}
+
+## Maintain system resources and running changefeeds
+
+When you are running more than 10 changefeeds on a cluster, it is important to monitor the [CPU usage]({% link {{ page.version.version }}/ui-overload-dashboard.md %}). A larger cluster will be able to run more changefeeds concurrently compared to a smaller cluster with more limited resources.
+
+{{site.data.alerts.callout_info}}
+{% include {{ page.version.version }}/cdc/changefeed-number-limit.md %}
+{{site.data.alerts.end}}
+
+To maintain a high number of changefeeds in your cluster:
+
+- Connect to different nodes to create each changefeed. The node on which you start the changefeed will become the _coordinator_ node for the changefeed job. The coordinator node acts as an administrator: keeping track of all other nodes during job execution and the changefeed work as it completes. As a result, this node will use more resources for the changefeed job. Refer to [How does an Enterprise changefeed work?]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) for more detail.
```

Comment:
can you add that this is also the case for resuming paused jobs, in addition to starting new ones?